ArduCopter/Parameters.cpp

```diff
@@ -1348,6 +1348,26 @@ void Copter::convert_tradheli_parameters(void)
         for (uint8_t i=0; i<table_size; i++) {
             AP_Param::convert_old_parameter(&singleheli_conversion_info[i], 1.0f);
         }
+
+        // convert to known swash type for setups that match
+        AP_Int16 *swash_pos_1, *swash_pos_2, *swash_pos_3, *swash_phang, *swash_type;
+        char pname[17];
+        enum ap_var_type ptype;
+        snprintf(pname, sizeof(pname), "H_SW_H3_SV1_POS");
+        swash_pos_1 = (AP_Int16 *)AP_Param::find(pname, &ptype);
```

Comment:
I think the snprintf can be skipped and instead the string can be directly put into the ::find() call's argument like this:

> swash_pos_1 = (AP_Int16 *)AP_Param::find("H_SW_H3_SV1_POS", &ptype);

The other place in Parameters.cpp where we do the "find" requires the snprintf because it dynamically sets the parameter name.